pep-0669.rst

```diff
@@ -186,13 +245,18 @@ Callback function arguments
 When an active event occurs, the registered callback function is called.
 Different events will provide the callback function with different arguments, as follows:
 
-* All events starting with ``PY_``:
+* ``PY_START`` and ``PY_RESUME``:
 
     ``func(code: CodeType, instruction_offset: int) -> DISABLE | Any``
 
-* ``C_CALL`` and ``C_RETURN``:
+* ``PY_RETURN`` and ``PY_YIELD``:
+
+    ``func(code: CodeType, instruction_offset: int, retval: object) -> DISABLE | Any``
 
-    ``func(code: CodeType, instruction_offset: int, callable: object) -> DISABLE | Any``
+* ``CALL``, ``C_RAISE`` and ``C_RETURN``:
+
+    ``func(code: CodeType, instruction_offset: int, callable: object, arg0: object | MISSING) -> DISABLE | Any``
+    If there are no arguments, ``arg0`` is set to ``MISSING``.
```

Comment:
```suggestion
    ``func(code: CodeType, instruction_offset: int, callable: object, arg0: object | MISSING) -> DISABLE | Any``

    If there are no arguments, ``arg0`` is set to ``MISSING``.
```

You probably want a line break here, so it doesn't get mashed together with the previous